README.md

```diff
@@ -56,6 +56,31 @@ This experiment is enabled by default and sessions between two Pat nodes (or oth
 
 For more information, see <https://github.com/la5nta/wl2k-go#gzip-experiment>.
 
+
+## Docker
+
+### Building
+
+To build Pat as a Docker image run the following:
+
+```
+docker build . -t pat:local
+```
+
+### Running the Docker image
+
+Create a `docker/data/config.json` file in it. This is the configuration file the Docker container will mount. Make sure the `http_addr` is set to `:8080` in `docker/data/config.json` otherwise the web interface won't function inside the container.
```

Comment:
I think it would be best not to recommend creating `config.json` manually. As long as the directory is valid and mounted, Pat should create this file on first startup with decent defaults (except the required http_addr mod).

Reply:
The web interface doesn't actually work unless you pre-create a `config.json`. The defaults don't work.